celery/bin/celery.py

```diff
@@ -75,7 +76,16 @@ def convert(self, value, param, ctx):
 APP = App()
 
 
-@with_plugins(entry_points().get('celery.commands', []))
+if sys.version_info >= (3, 10):
+    _PLUGINS = entry_points(group='celery.commands')
+else:
+    try:
+        _PLUGINS = entry_points().get('celery.commands', [])
+    except AttributeError:
+        _PLUGINS = entry_points().select(group='celery.commands')
```

Comment:
This will fail on `importlib_metadata<3.6`, so I suggest bumping to `importlib_metadata>=3.6` or use [backports.entry_points_selectable](https://pypi.org/project/backports.entry_points_selectable).

Reply:
done